src/CSharp/MetadataWebApi/MetadataWebApi.Tests/MetadataApiFactoryTests.cs

```diff
@@ -90,39 +103,33 @@ public static void MetadataApiFactory_CreateMetadataApi_Creates_Instance_If_Inva
         }
 
         [Fact]
-        public static void MetadataApiFactory_CreateMetadataApi_Throws_If_No_Token_Configured()
+        public void MetadataApiFactory_CreateMetadataApi_Throws_If_No_Token_Configured()
         {
             // Arrange
-            Mock<MetadataApiFactory> mock = new Mock<MetadataApiFactory>();
-
-            mock.CallBase = true;
-            mock.Setup((p) => p.GetConfigSetting("Token")).Returns(string.Empty);
+            var configuration = new ConfigurationBuilder()
+                .AddInMemoryCollection(inMemoryConfigEmpty)
```

Comment:
I prefer without but the code here is inconsistent. _ was originally used by MS code to minimize the possibility of collisions with usercode.